### ISSUE TITLE
is_leap tests are sensitive to time zone

### ISSUE THREAD

**Issue by mmastrac** [denoland/deno_std#3299]
**Describe the bug**

The `is_leap` function works, but the test cases construct a date that is sensitive to time zones.

https://github.com/denoland/deno_std/commit/8cf6b45e2cdb202235945928dcec641d390ae579

The test should probably use a date other than Jan 1 to ensure that the year is consistent in all time zones, for example:

```
> new Date("1972-01-01").getFullYear()
1971
> new Date("1972-02-01").getFullYear()
1972
```

**Comment by tetraphobia** [comment]
I believe this is mentioned in #3294 and would be solved by #3295

**Comment by kt3k** [comment]
fixed by #3295
